### ISSUE TITLE
TravisCI failures

### ISSUE THREAD

**Issue by asmallteapot** [plangrid/ReactiveLists#116]
Travis appears to be [having problems building master](https://travis-ci.org/plangrid/ReactiveLists/jobs/371770354#L792):

```
❌  /Users/travis/build/plangrid/ReactiveLists/Sources/TableViewDriver.swift:113:47: value of type '[IndexPath]' has no member 'compactMap'
        let indexPathsAndViewModelsToReload = visibleIndexPaths.compactMap { indexPath in
                                              ^~~~~~~~~~~~~~~~~ ~~~~~~~~~~
```

**Comment by anayini** [comment]
Hm could be because `compactMap` is only available on 9.3+?

**Comment by jessesquires** [comment]
lol. yeah. I haven't been maintaining travis very well. 

I'm planning to devote some love to this project on Friday, if possible
